terraformInstall.sh

```diff
@@ -1,34 +1,47 @@
 #!/bin/bash
 
 ## CONDITIONAL 1
-echo "Installing prerequisites..."
-sudo apt-get update && sudo apt-get install -y gnupg software-properties-common
+# Update package lists
+echo "Installing Prerequisites..."
+if sudo apt update; then
```

Comment:
sudo apt update is the resolution/remediation, not something you test. Try flipping it to where `apt-cache show PACKAGE` is the check and `sudo apt update` is the action in the else.

Reply:
I added the apt-cache show revision as you suggested.